watson/cli.py

```diff
@@ -265,14 +285,36 @@ def status(watson):
     ))
 
 
+_SHORTCUT_OPTIONS = ['year', 'month', 'week', 'day']
+
+
 @cli.command()
-@click.option('-f', '--from', 'from_', type=Date,
+@click.option('-f', '--from', 'from_', cls=MutuallyExclusiveOption, type=Date,
```

Comment:
Can you explain, why you don't use click's [feature switches](http://click.pocoo.org/5/options/#feature-switches) here?